### PR TITLE
srcml 0.9.5 (new formula)

### PR DESCRIPTION
**Pull request by yijunyu**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/srcml.rb [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+  url "http://sead1.open.ac.uk/srcml/srcml-0.9.5.tar.gz"
+  sha256 "55dd2115548e270724af4251187343656d2dfda0e7d372fee15ae27262e3fa8e"
+
+  bottle do
+    root_url "http://sead1.open.ac.uk/srcml"
+    cellar :any
+    rebuild 1
+    sha256 "4ccf28e8a31f43dc80d0eb0b9947db18215803804b590506fd52403f1bc28f9d" => :sierra
+    sha256 "406e548e4cd91f19b67bb764c16adb7b6467bf8fde1a791f5c001fc9a7925ae3" => :el_capitan
+  end
```
Comment (bfontaine) [inline Formula/srcml.rb]: Please remove this block; our bot will take care of it.

File: Formula/srcml.rb [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+  depends_on "boost" => :build
+  depends_on "libxml2" => :build
+  depends_on "libarchive"
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+  end
+
+  patch :DATA
```
Comment (bfontaine) [inline Formula/srcml.rb]: Please submit this patch to upstream.

File: Formula/srcml.rb [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+
+  test do
+    (testpath/"Hello.java").write <<-EOS
+int
```
Comment (bfontaine) [inline Formula/srcml.rb]: Please use `<<-EOS.undent` above and indent this line.

File: Formula/srcml.rb [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+    EOS
+    exec "#{bin}/srcml", "Hello.java", "-o", "/dev/null"
```
Comment (bfontaine) [inline Formula/srcml.rb]: Use `system`, not `exec`.

**Review by bfontaine — CHANGES_REQUESTED** [review]

**Comment by dunn** [comment]
Any update?

**Comment by ilovezfs** [comment]
@yijunyu Thanks for the PR. Feel free to re-open once the requested changes have been made.

**Pull request closed without merging**